Use the GUI check in the GUI_SRC_$(VERSION) directory.

Index: salome/SMESH_SRC_5.1.3/configure.ac
===================================================================
--- salome.orig/SMESH_SRC_5.1.3/configure.ac
+++ salome/SMESH_SRC_5.1.3/configure.ac
@@ -259,21 +259,22 @@
 echo ---------------------------------------------
 echo
 
-CHECK_GUI_MODULE
+CHECK_SALOME_GUI
+CHECK_CORBA_IN_GUI
 
 gui_ok=no
-if test "${SalomeGUI_need}" != "no" -a "${FullGUI_ok}" = "yes" ; then 
+if test "${SalomeGUI_need}" != "no" -a "${CORBA_IN_GUI}" = "yes" ; then 
   gui_ok=yes
 fi
 
 AM_CONDITIONAL(SMESH_ENABLE_GUI, [test "${gui_ok}" = "yes"])
 
 if test "${SalomeGUI_need}" == "yes"; then
-  if test "${FullGUI_ok}" != "yes"; then
+  if test "${CORBA_IN_GUI}" != "yes"; then
     AC_MSG_WARN(For configure SMESH module necessary full GUI!)
   fi
 elif test "${SalomeGUI_need}" == "auto"; then
-  if test "${FullGUI_ok}" != "yes"; then
+  if test "${CORBA_IN_GUI}" != "yes"; then
     AC_MSG_WARN(Full GUI not found. Build will be done without GUI!)
   fi
 elif test "${SalomeGUI_need}" == "no"; then
Index: salome/SMESH_SRC_5.1.3/build_configure
===================================================================
--- salome.orig/SMESH_SRC_5.1.3/build_configure
+++ salome/SMESH_SRC_5.1.3/build_configure
@@ -46,6 +46,14 @@
 #fi
 
 ########################################################################
+# Test if the GUI_ROOT_DIR is set correctly
+
+if test ! -d "${GUI_ROOT_DIR}"; then
+    echo "failed : GUI_ROOT_DIR variable is not correct !"
+    exit
+fi
+
+########################################################################
 # Test if the MED_ROOT_DIR is set correctly
 
 if test ! -d "${MED_ROOT_DIR}"; then
@@ -79,6 +87,7 @@
 
 aclocal -I adm_local/unix/config_files \
         -I ${KERNEL_ROOT_DIR}/salome_adm/unix/config_files \
+        -I ${GUI_ROOT_DIR}/adm_local/unix/config_files \
         -I ${MED_ROOT_DIR}/adm_local/unix/config_files \
 	-I ${GEOM_ROOT_DIR}/adm_local/unix/config_files || exit 1
 
